app/uk/gov/hmrc/bindingtariffclassification/model/Sequence.scala

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.bindingtariffclassification.model
+
+case class Sequence(name: String, value: Long)
```

Comment:
I believe we need this only for cases.
So, there is not need of storing a `name`

Reply:
cases could be BTI and liability  wich could be different as well as maybe events or other elements

Reply:
Good point @pllaurado